lib/files.cf

```diff
@@ -436,33 +438,43 @@ bundle edit_line set_variable_values_ini(tab, sectionName)
       # Be careful if the index string contains funny chars
       "cindex[$(index)]" string => canonify("$(index)");
 
+      # Escape the value so that it can be regexed
+      "ctab[$(index)]" string => escape("$($(tab)[$(sectionName)][$(index)])");
+
   classes:
       "edit_$(cindex[$(index)])"     not => strcmp("$($(tab)[$(sectionName)][$(index)])","dontchange"),
       comment => "Create conditions to make changes";
 
-  field_edits:
-
-      # If the line is there, but commented out, first uncomment it
-      "#+\s*$(index)\s*=.*"
-      select_region => INI_section(escape("$(sectionName)")),
-      edit_field => col("\s*=\s*","1","$(index)","set"),
-      if => "edit_$(cindex[$(index)])";
-
-      # match a line starting like the key something
-      "\s*$(index)\s*=.*"
-      edit_field => col("\s*=\s*","2","$($(tab)[$(sectionName)][$(index)])","set"),
-      select_region => INI_section(escape("$(sectionName)")),
-      classes => results("bundle", "set_variable_values_ini_not_$(cindex[$(index)])"),
-      if => "edit_$(cindex[$(index)])";
+      "pass2" expression => "pass1";
```

Comment:
Ewwwww. Why are you forcing a specific pass restriction?

Presumably, something isn't yet defined or hasn't been completed. Or, not able to select region until the second pass?

Reply:
Exactly. If it's on the first pass, if the section doesn't exist yet, it cannot select the region, and fails. It won't reevaluate at the 2nd pass

Reply:
Right, but since the promise to insert the section is before the promise to set key values for the region, it should be able to select it during the same pass.